### ISSUE TITLE
Recurring Tasks - still confused

### ISSUE THREAD

**Issue by Nilpointer** [rails/solid_queue#420]
Hi,

I am still unsure how to manage reccuring tasks. 

- How do I delete them? ( I don't understand the provided solutions - sorry)
- How can I pause them? Can I?
- How can I update the schedule value at runtime with no downtime?

Can I simply edit the solid_queue_recurring_tasks schedule value or does this cause unwanted sideeffects?

Regards 
Andreas

**Comment by rosa** [comment]
> How do I delete them? ( I don't understand the provided solutions - sorry)

You delete them from your `recurring.yml` configuration (or another file if you are configuring them elsewhere). 

> How can I pause them? Can I?

No, pausing them is not supported. You'd need to remove them and then re-add them. 

> How can I update the schedule value at runtime with no downtime?

You'd need to have a no-downtime deploy as you need to update your configuration and deploy your code. 

> Can I simply edit the solid_queue_recurring_tasks schedule value or does this cause unwanted sideeffects?

This won't work in general because in most cases the scheduler will have the tasks already loaded from when it started, and then when it restarts, it'll read the configuration and would load any task there again, overwriting your changes in `solid_queue_recurring_tasks`.

**Comment by rosa** [comment]
> But how? If I remove the development node from my schedule.yml, restart the application - the entries in the solid_queue_recurring_tasks table are still there.

Yes, sorry, they're there but it doesn't mean they're running. See https://github.com/rails/solid_queue/issues/382. I'll change that so they get deleted as well when the scheduler reloads again. I just haven't had time this week.

**Comment by Nilpointer** [comment]
Is there a way to inspect/query what jobs are currently scheduled?

**Comment by rosa** [comment]
@Nilpointer just checking the contents of `recurring.yml`. I'm going to fix this shortly, in any case.

**Comment by rosa** [comment]
This one should be now handled by #433.
